views/partials/email_partials/report_what_to_do_next.hbs

```diff
@@ -7,14 +7,14 @@
       </td>
     </tr>
 
-    {{#each steps }}
+    {{#each steps}}
       <tr class="email-wtd">
         <td valign="top" width="60" style="width: 60px; padding-top: 15px;">
-          <img width="45" src="{{ ../SERVER_URL }}/img/email_images/wtd-{{ idx }}.png" alt="{{ idx }}" />
+          <img width="45" src="{{ srcDomain }}/img/email_images/wtd-{{ idx }}.png" alt="{{ idx }}" />
         </td>
         <td valign="top" style="padding-top: 10px; font-family: sans-serif;">
           <p class="body-copy medium" style="color: #333333; font-weight: 400; font-size: 16px; padding: 0px; margin: 0px; ">
-            {{ subhead }}
+            {{ subhead }}{{ SERVER_URL }}
```

Comment:
Curious... What is the `{{ SERVER_URL }}` for?

Reply:
Nothing! Thanks for catching this.